### ISSUE TITLE
Hey all - meeting? 

### ISSUE THREAD

**Issue by alexander-moore** [moorea1/DS501_Project1#1]
Hi everyone

Let me know if you see this issue. I like github issues because they are a good way to organize our work.

I want to propose meeting early this week, and we each bring in 1-2 proposals for what our angle for the project is. We would each check out the data set and assignment and bring in a couple ideas for the data and methods we would use.

Let me know what you think, if + when you want to meet, etc

Alex

**Comment by ebprihar** [comment]
I have seen the issue. Here’s a when2meet to make meeting easier this week:

https://www.when2meet.com/?8148840-1ko3y

**Comment by mbarger1** [comment]
I put my availability in the when2meet. As a heads up, I'm going to a wedding this week and won't be available for in-person work from Thursday onwards. I should have some time to work on things while I'm there, but I'd like to have a clear definition of the project before I leave Thursday afternoon so that I can contribute meaningfully.

**Comment by ebprihar** [comment]
Based on the when2meet, I propose we meet tomorrow at 6:00pm in the library and get started. All in favor say something indicative of that.

**Comment by mbarger1** [comment]
Aye aye captain

**Comment by alexander-moore** [comment]
Sounds good - is the library next to fuller?

**Comment by mbarger1** [comment]
Yeah, it's right by fuller. How long do we want to meet for? I can reserve a tech suite

**Comment by alexander-moore** [comment]
Id say 1.5 hours tops. Ideally we propose a topic and divide up work then can start working independently from there

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by mbarger1** [comment]
Awesome. I booked tech suite 112A for 6:00-7:30, in case we want it the whole time.

**Comment by alexander-moore** [comment]
Nice, thanks!

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by qh2150** [comment]
Sounds good, I'll see everyone tomorrow.
